nextflow.config

```diff
@@ -160,8 +168,28 @@ process.ext.override_configured_container_registry = true
 // Nextflow plugins
 plugins {
     id 'nf-validation@1.1.3' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-iridanext@0.3.0'
 }
 
+iridanext {
```

Comment:
This is a small note that doesn't impact how it runs, but I've been keeping the IRIDA Next configuration in a separate file in `config/irida.config` to make it easier to find and update. I'm wondering if you could do the same. And then have an `include` statement in the main `nextflow.config` file to import it?

Reply:
Updated in a432a04743ee1bc20ae1f542f753c5061be406ad